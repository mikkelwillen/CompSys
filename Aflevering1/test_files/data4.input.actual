test_files/data4.input: ASCII text
