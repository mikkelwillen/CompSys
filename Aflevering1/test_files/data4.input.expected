test_files/data4.input: data
